src/sql-parser/tests/testdata/literal

```diff
@@ -260,7 +277,11 @@ Array([Array([Value(Number("1")), Value(Number("2"))]), Array([Value(Number("3")
 parse-scalar
 ARRAY[3]]
 ----
-Array([Value(Number("3"))])
```

Comment:
This is maybe an error in the test input? It looks like it was supposed to test a success case.